local/install_deps_linux.bash

```diff
@@ -121,6 +121,11 @@ sudo apt-get install -y \
     unzip \
     xvfb
 
+# Only install the google cloud SDK via apt if it's missing.
+if ! which gcloud > /dev/null 2>&1; then
+  sudo apt-get install -y google-cloud-sdk
```

Comment:
This looks incorrect since this is already done at line 161, but not running due to bound by 
if [ ! $only_reproduce ]; then condition.

Maybe take out
  # Install gcloud dependencies.
  if gcloud components install --quiet beta; then	  if gcloud components install --quiet beta; then
    gcloud components install --quiet \	    gcloud components install --quiet \
        app-engine-go \	        app-engine-go \
        app-engine-python \	        app-engine-python \
        app-engine-python-extras \	        app-engine-python-extras \
        beta \	        beta \
        cloud-datastore-emulator \	        cloud-datastore-emulator \
        pubsub-emulator	        pubsub-emulator
  else	  else
    # Either Cloud SDK component manager is disabled (default on GCE), or google-cloud-sdk package is	    # Either Cloud SDK component manager is disabled (default on GCE), or google-cloud-sdk package is
    # installed via apt-get.	    # installed via apt-get.
    sudo apt-get install -y \	    sudo apt-get install -y \
        google-cloud-sdk-app-engine-go \	        google-cloud-sdk-app-engine-go \
        google-cloud-sdk-app-engine-python \	        google-cloud-sdk-app-engine-python \
        google-cloud-sdk-app-engine-python-extras \	        google-cloud-sdk-app-engine-python-extras \
        google-cloud-sdk \	        google-cloud-sdk \
        google-cloud-sdk-datastore-emulator \	        google-cloud-sdk-datastore-emulator \
        google-cloud-sdk-pubsub-emulator	        google-cloud-sdk-pubsub-emulator
  fi	  fi

out of 
only_reproduce

Reply:
Good point. Addressed in #1038